ocaml/xapi/repository_helpers.ml

```diff
@@ -875,38 +875,44 @@ let remove_repo_conf_file repo_name =
   in
   Unixext.unlink_safe path
 
-let write_yum_config ?(source_url = None) binary_url repo_name =
+let write_yum_config ?(source_url = None) ?(repo_gpgcheck = true) binary_url
+    repo_name gpgkey_name =
```

Comment:
named arguments is better since the arguments share type and can be easily switched